frontend/src/components/pages/profile/Profile.tsx

```diff
@@ -1,10 +1,186 @@
-import * as React from 'react';
+import * as React from "react";
+import Button from "@material-ui/core/Button";
+import { createStyles } from "@material-ui/core/styles";
+import FilledInput from "@material-ui/core/FilledInput";
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import Input from "@material-ui/core/Input";
+import InputLabel from "@material-ui/core/InputLabel";
+import { makeStyles } from "@material-ui/core/styles";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import { PersonProps } from "../../../services/mock_profile_backend";
+import { USER_ID } from "../../../utils/temporary_testing_consts";
+import { ProfileHandlerService } from "../../../services/profile_handler_service";
+import { MockProfileBackendService } from "../../../services/mock_profile_backend";
+import { ServiceContext } from "../../contexts/contexts";
+import { ServiceHandlers } from "../../contexts/contexts";
+import { Theme } from "@material-ui/core/styles";
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+      flexWrap: 'wrap',
+      marginTop: '100px',
+      marginLeft: theme.spacing(4),
+    },
+    '& input:invalid + fieldset': {
+      borderColor: 'red',
```

Comment:
design nit (feel free to fix this later): Conveying error states with color alone isn't enough, since that might not be obvious to people with certain forms of color blindness.

Reply:
Super valid point for accessibility! Do you think that the most appropriate way to do this would be using an aria-label?

Reply:
Aria-labels are good too! But color-blind users wouldn't be using a screenreader. The usual solution is to add a label under the component as well, or some sort of icon indicating that there was an error (like an arrow pointing to the field that is incorrect)